### PR TITLE
docs(readme): interaction test script corrected

### PR DESCRIPTION
**Pull request by sanketshevkar**
Signed-off-by: User <shevkar.sanket@gmail.com>

### Changes

- Script for user interaction testing needed to be corrected in the readme file.

### Related Issues
- Issue #14 

### Author Checklist
- [ ] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [ ] Vital features and changes captured in unit and/or integration tests
- [ ] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [ ] Merging to `master` from `fork:branchname`
- [ ] Manual accessibility test performed
    - [ ] Keyboard-only access, including forms
    - [ ] Contrast at least WCAG Level A
    - [ ] Appropriate labels, alt text, and instructions

### CODE REVIEW

File: README.md
Comment (rkotangoor) [inline README.md:42]: we need to be inside storybook folder to run this command?, you might want to include that too. 
This will be:
```
cd packages/storybook
npm run test:e2e
```

Comment (sanketshevkar) [reply README.md:42]: I had added it to the step above it. Still I'll add it to this step too as it'll be more clear.

**Review by rkotangoor — APPROVED** [review]
lgtm, will wait for one of the maintainers to review and approve.

**Review by mttrbrts — APPROVED** [review]

**Pull request merged**
